kratos/includes/kratos_application.h

```diff
@@ -141,6 +146,25 @@ class KRATOS_API(KRATOS_CORE) KratosApplication {
 
     void RegisterKratosCore();
 
+    /**
+     * @brief This method is used to unregister common components of the application.
+     * @details This method is used to unregister common components of the application. 
+     * The list of unregistered components are the ones exposed in the common KratosComponents interface:
```

Comment:
For consistency, I would suggest to use "deregister" everywhere, rather than mixing it with "unregister".
```suggestion
     * @brief This method is used to deregister common components of the application.
     * @details This method is used to deregister common components of the application. 
     * The list of deregistered components are the ones exposed in the common KratosComponents interface:
```

Reply:
Agree. It will be done like in #12281, just wanted to avoid having more changes than the necessary here as I got a lot of comments about renaming in the other PR